### ISSUE TITLE
Drag and Drop into Reaper VST only working from QT based file managers.

### ISSUE THREAD

**Issue by OzFalcon** [robbert-vdh/yabridge#303]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

Only QT based file managers are able to drag and drop into VST

### What did you expect to happen?

Other (GTK) based file managers to also be able to drag and drop.

### What actually happened?

QT based file managers work as expected.
GTK based file managers do not work as expected.

### Operating system

Debian 12.4

### How did you install yabridge?

Manualy into ~/.local/share/yabridge/

### yabridge version

5.1.0

### yabridgectl version

5.1.0

### Wine version

Wine 9 development branch

### Plugin

Sitala-1.0-Setup-64.msi

### Plugin type

VST2

### Plugin architecture

64-bit

### Host

Reaper 7.0.8

### Desktop environment or WM

XFCE 4.18

### GPU model

NVidia RTX 3060

### GPU drivers and kernel

NVidia driver 525.147.05

### Anything else?

As far as I can tell, QT based file managers work.
I'm not sure why other file managers do not work.

Free version of plugin https://decomposer.de/sitala/releases/Sitala-1.0-Setup-64.msi

Here are the results of drag and drop into the windows VST.

Nemo Failed
Thunar Failed
Dolphin Success
Nautilus Failed
PCManFM Failed
PCManFM-Qt Success

Correct operation with Dolphin File Manager
https://private-user-images.githubusercontent.com/13824723/296635339-86aa73f0-1595-408d-b74a-464c46b56a6b.png
https://private-user-images.githubusercontent.com/13824723/296635347-d13e7498-63e1-4b95-b923-387752e6b3e0.png

**Comment by OzFalcon** [comment]
Screenshot of Dolphin Drag and Drop working.
![Dolphin Drag](https://github.com/robbert-vdh/yabridge/assets/13824723/86aa73f0-1595-408d-b74a-464c46b56a6b)
![Dolphin Drop](https://github.com/robbert-vdh/yabridge/assets/13824723/d13e7498-63e1-4b95-b923-387752e6b3e0)

**Comment by wastee** [comment]
Hi. I cannot drag file from Dolphin or Nemo/Nautilus to the VST window. I can drag from the VST window to REAPER, but not the other way around. Maybe it's related to Wayland.

**Comment by wastee** [comment]
Yes, i can confirm Dolphin (QT) can drag and drop files into wine VST on Linux REAPER under X11.

I did more test

## what can be:
- X11 Dolphin can drag and drop files into wine VST in Linux REAPER.
- Wine REAPER can drop files into wine VST in Linux REAPER.
- Wine Explorer can drop files into wine VST in Linux REAPER.

## what not can be:

- In Wayland and X11, Linux REAPER cannot drag files from REAPER into wine VST.
- Wayland and all Linux file managers cannot drag and drop files into wine VST in Linux REAPER.
